hermes-common/src/main/resources/hermesMessage.avsc

```diff
@@ -0,0 +1,18 @@
+{
+    "namespace": "pl.allegro.tech.hermes",
```

Comment:
we don't relay on this, this file is only used in one unit test and it was added much for illustrative purpose. We do avro de/serialization at lower level and we don't reference to hermes schema file. See AvroMessageContentWrapper class.